packages/@lwc/integration-karma/test/light-dom/scoped-slot/index.spec.js

```diff
@@ -61,7 +61,7 @@ describe('scoped slots', () => {
         // For standard slot content, "slot" attribute goes directly on the element unlike scoped
         //  slots where the attribute goes on the template tag
         expect(child.querySelector('.slotname3').innerHTML).toBe(
-            `${vFragBookend}<p slot="slotname3">MLB</p>${vFragBookend}`
+            `${vFragBookend}<p>MLB</p>${vFragBookend}`
```

Comment:
This is an observable change. Should we do it through API versioning?

Reply:
Suggestion from PR review today - Run nucleus tests through all tests, even the non-white listed and verify that it doesn't break anything. This will help determine if we need to use API versioning.

Reply:
See https://github.com/salesforce/lwc/pull/3883#issuecomment-1853173719